docs/UserGuide.md

```diff
@@ -281,14 +284,44 @@ If your changes to the data file make its format invalid, FoodWhere will discard
 | **help**  | `help`                                                                                                       |
 | **clear** | `clear`                                                                                                      |
 | **radd**  | `radd s/STALL_INDEX d/DATE c/CONTENT r/RATING [t/TAGS]…` <br> e.g., `radd s/3 d/20/09/2022 c/Great food! r/4`|
-| **rdel**  | `rdel INDEX`                                                                                                 |
-| **redit** | `redit INDEX [d/DATE] [c/CONTENT] [r/RATING] [t/TAGS]…` <br> e.g., `redit d/20/09/2022 c/Great food!`        |
+| **rdel**  | `rdel REVIEW_INDEX`                                                                                                 |
+| **redit** | `redit REVIEW_INDEX [d/DATE] [c/CONTENT] [r/RATING] [t/EDIT_TAG]…` <br> e.g., `redit 1 d/20/09/2022 c/Great food!`      |
 | **rlist** | `rlist`                                                                                                      |
-| **rfind** | `rfind n/NAME_KEYWORD [MORE_KEYWORDS]… t/TAG_KEYWORD [MORE_KEYWORDS]…` <br> e.g., `rfind n/eatery`           |
-| **rsort** | `rsort CRITERION` <br> e.g., `rsort rating`                                                                  |                                                                   
-| **sadd**  | `sadd n/NAME a/ADDRESS [t/TAGS]…` <br> e.g., `sadd n/John Chicken Rice a/Blk 123 Bedok South t/veryNice`     |
-| **sdel**  | `sdel INDEX`                                                                                                 |
-| **sedit** | `sedit INDEX [n/NAME] [a/ADDRESS] [t/TAGS]…` <br> e.g., `sedit n/John Chicken Rice a/Blk 123 Bedok South`    |
+| **rfind** | `rfind n/KEYWORD [KEYWORD]… t/KEYWORD [KEYWORD]…` <br> e.g., `rfind n/eatery`                                |
+| **rsort** | `rsort CRITERION` <br> e.g., `rsort rating`                                                                  |
+| **sadd**  | `sadd n/NAME a/ADDRESS [t/TAGS]…` <br> e.g., `sadd n/John Chicken Rice a/Blk 123 Bedok South`                |
+| **sdel**  | `sdel STALL_INDEX`                                                                                           |
+| **sedit** | `sedit STALL_INDEX [n/NAME] [a/ADDRESS] [t/EDIT_TAG]…` <br> e.g., `sedit 1 n/John Chicken Rice`                  |
 | **slist** | `slist`                                                                                                      |
-| **sfind** | `sfind n/NAME_KEYWORD [MORE_KEYWORDS]… t/TAG_KEYWORD [MORE_KEYWORDS]…` <br> e.g., `rfind n/eatery`           |
+| **sfind** | `sfind n/KEYWORD [KEYWORD]… t/KEYWORD [KEYWORD]…` <br> e.g., `rfind n/eatery`                                |
```

Comment:
@clarence-chew you seem to have missed out on the first issue from #282. Fix `rfind` command keyword in example to the correct, `sfind` command keyword.